website/addons/wiki/templates/edit.mako

```diff
@@ -8,38 +8,186 @@
             <%include file="wiki/templates/nav.mako"/>
             <%include file="wiki/templates/toc.mako"/>
         </div>
-         <div class="col-sm-9">
-                 <%include file="wiki/templates/status.mako"/>
-            <form action="${urls['web']['edit']}" method="POST">
+        <div class="col-sm-9">
+            <%include file="wiki/templates/status.mako"/>
+            <form id="wiki-form" action="${urls['web']['edit']}" method="POST">
                 <div class="form-group wmd-panel">
-                    <div id="wmd-button-bar"></div>
-                    <textarea class="form-control wmd-input" rows="25" id="wmd-input" name="content" data-bind="value: wikiText"></textarea>
+                    <div class="row">
+                        <div class="col-sm-8">
+                             <p>
+                                 <em>Changes will be stored but not published until
+                                 you click "Save."</em>
+                             </p>
+                            <div id="wmd-button-bar"></div>
+                        </div>
+                        <div class="col-sm-4">
+                            <ul class="list-inline" data-bind="foreach: activeUsers" style="float: right">
+                                <!-- ko ifnot: id === '${user_id}' -->
```

Comment:
Instead of using mako to get user_id, you can attach the current user's ID --which you can get from `window.contextVars.user.id`--to the ViewModel